cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -7,10 +7,16 @@
 from core.tests.mommy_utils import make_recipe
 from cla_auditlog.models import AuditLog
 from complaints.models import Complaint
-from legalaid.models import Case, EODDetails
+from legalaid.models import Case, EODDetails, PersonalDetails
 
 
 class TasksTestCase(TestCase):
+    """
+    Currently in Django v1.7 these tests anything with an M2M relationship will not be tested correctly
+    Due to the DB of the tests not being a persisted.
+    Updating to at least Django v1.8 should fix this.
```

Comment:
What is the current state of these tests, then? Passing, but they would always pass, even if the code hadn't been improved?
What will change when the django version is upgraded? That they will continue to pass, but would now no longer be liable to false positives?